src/persist-client/tests/trace/since_upper

```diff
@@ -0,0 +1,54 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# since and upper start a minimum ts
+since-upper
+----
+[0][0]
+
+# A [0,1) batch advances the upper to [1]
+insert
+0 1 0 0
```

Comment:
fwiw, I found most of the datadriven tests pretty readable without digging into their syntax, but this one tripped me up

Reply:
good to know! i can totally see it. any thoughts on what to do instead?